src/api/ledger/balance-sheet.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const _ = require('lodash');
+const utils = require('./utils');
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+const convertErrors = utils.common.convertErrors;
+
+function formatBalances({balances, obligations, assets}) {
+  const result = {};
+
+  if (!_.isUndefined(balances)) {
+    result.balances = balances;
+  }
+  if (!_.isUndefined(obligations)) {
+    result.obligations = obligations;
+  }
+  if (!_.isUndefined(assets)) {
+    result.assets = assets;
+  }
+
+  return result;
+}
+
+function getBalanceSheetAsync(account, options, callback) {
```

Comment:
All the methods use `account` currently, but probably would be better to change them to `address` (in another pull request).